atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/CassandraCliParser.java

```diff
@@ -31,9 +31,10 @@ public static int parseSystemAuthReplicationFromCqlsh(String output) throws Ille
         try {
             for (String line : output.split("\n")) {
                 if (line.contains("system_auth")) {
-                    Matcher matcher = Pattern.compile("^.*\\{\"replication_factor\":\"(\\d+)\"\\}$").matcher(line);
+                    String replicationRegex = "^.*('|\")replication_factor('|\"):( '|\")(\\d+)('|\")\\}$";
+                    Matcher matcher = Pattern.compile(replicationRegex).matcher(line);
                     matcher.find();
-                    return Integer.parseInt(matcher.group(1));
+                    return Integer.parseInt(matcher.group(4));
```

Comment:
We should make this a named group so we can easily see what it references now that there is more than one group.